source/user-manual/capabilities/syscollector.rst

```diff
@@ -422,7 +422,7 @@ In Elasticsearch the fields will be saved as ``data.type.value``. For example, f
 +----------------------+----------------------------------------------------------------------------------------------------------------------+----------------------------------+
 | **Operating System** | architecture, name, version, codename, major, minor, build, platform, sysname, release, release_version              | data.os.codename                 |
```

Comment:
Add the `os_release` field to this table, and the other places it has to be added.